src/Accessibility.ts

```diff
@@ -108,6 +110,32 @@ export interface Accessibility {
    * `null` indicates there is no media, `undefined` or missing property indicates unknown.
    */
   media?: ArrayLike<Media> | null;
+  /**
+   * `true` if the facility provides mobility equipment e.g. foldable wheelchairs or crutches
+   */
+  providesMobilityEquipment?: boolean;
+  /**
+   * Describes the pick up or drop off zone
+   */
+  pickUpOrDropOffZone?: Ground | null;
```

Comment:
When reading `pickUpOrDropOffZone`. What do you think about changing this to `groundConditionsAtPickupOrDropoffZone`, even if it's long? At least that would be concise :)

Reply:
Yes, sounds good. Will do.

Reply:
renamed